gradle.properties

```diff
@@ -1,3 +1,3 @@
 org.gradle.daemon=true
 org.gradle.jvmargs=-Xms256m -Xmx1024m
-archash=fcc8d5bf44d872ec60eee557f6db16599ed267c2
+archash=46d1f7c89424a7abf0d7c51cd24f2a185ba98337
```

Comment:
Why change version of Arc?

Reply:
Dunno, happened on its own. Probably something with Gradle.

Reply:
And by the power of console, I reverted this change.